namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/calculate-release-dates-api-topic.tf

```diff
@@ -0,0 +1,14 @@
+resource "kubernetes_secret" "calculate-release-dates-api-topic" {
+  metadata {
+    name      = "hmpps-domain-events-topic"
+    namespace = var.namespace
+    # Remove when namespace has been migrated
```

Comment:
I've commented out all references to namespaces that haven't been migrated yet and instead inject into current namespace